docs_versioned_docs/version-ros2jazzy/components/yaml/attachments/a300/amp_sensor_arch.mdx

```diff
@@ -0,0 +1,35 @@
+<table>
```

Comment:
Intent the HTML block.

Reply:
See https://github.com/clearpathrobotics/cpr-documentation/pull/404#discussion_r2161996264